jaeger-zipkin/src/test/java/io/jaegertracing/zipkin/ZipkinSenderTest.java

```diff
@@ -137,14 +141,35 @@ public void testFlushSendsSpan() throws Exception {
     zipkin2.Span actualSpan = traces.get(0).get(0);
     JaegerSpanContext context = expectedSpan.context();
 
-    assertEquals(context.getTraceId(), HexCodec.lowerHexToUnsignedLong(actualSpan.traceId()));
+    assertEquals(context.getTraceIdLow(), HexCodec.lowerHexToUnsignedLong(actualSpan.traceId()));
+    assertEquals(context.getTraceIdHigh(), 0L);
     assertEquals(context.getSpanId(), HexCodec.lowerHexToUnsignedLong(actualSpan.id()));
     assertEquals(context.getParentId(), 0L);
     assertNull(actualSpan.parentId());
     assertEquals(expectedSpan.getOperationName(), actualSpan.name());
     assertEquals(tracer.getServiceName(), actualSpan.localServiceName());
   }
 
+  @Test
+  public void testAppend128BitTraceId() throws Exception {
```

Comment:
It seems the only place 64bit logic should be tested is in the trace ID generation (i.e. tests for `tracer.startspan()`). All other tests, especially those that deal with external formats (codecs and senders) should probably be only operating on 128bit (since it implicitly tests 64bit conversion as well).

But it's probably going to involve a lot of test changes, please lmk.

Reply:
The changes would have been to big. I prefered to isolate the testing of 128bit (sometimes duplicating the test of 64 bit). So that I'm sure that the 64 bit test coverage stays the same. At some point refactoring the tests might make sense (maybe if deprecating 64bits traceid).
But until 64 bit traceid are considered fully supported, I would rather continue testing everything on 64bit and test 128 support in other/more tests.